tasks/request_files/request_files.py

```diff
@@ -326,172 +361,91 @@ def process_granule(s3: BaseClient,
         if not a_file[FILE_SUCCESS_KEY]:
             any_error = True
             # If this is reached, that means there is no entry in the db for file's status.
-            post_status_for_file_to_queue(
-                job_id, granule_id, os.path.basename(a_file[FILE_KEY_KEY]),
-                a_file[FILE_KEY_KEY],
-                a_file[FILE_DEST_BUCKET_KEY],
-                ORCA_STATUS_FAILED,
+            shared_recovery.update_status_for_file(
```

Comment:
Not sure this is needed, @AndrewDorn you and I should talk through this logic.

Reply:
Only needed if logic above is not modified.

Reply:
removed  shared_recovery.update_status_for_file()